src/data/full-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const fullQuiz = [
+  {
+    Question: "In vim editor, which mode helps us to copy, paste and delete the text?",
+    Answer: "visual mode",
+    Distractor1: "yanking mode",
+    Distractor2: "insert mode",
+    Distractor3: "normal mode",
+    Explanation:
+      "Visual Mode in vim helps the user to copy, paste and delete the text as like a normal editor.",
```

Comment:
I think we should remove the word "as" at the end of that sentence so it reads a little bit better. 

Proposed fix: 
Visual Mode in Vim helps the user to copy, paste and delete the text like a normal editor.